configure

```diff
@@ -3475,6 +3475,7 @@ ogg_demuxer_select="dirac_parse"
 ogv_muxer_select="ogg_muxer"
 opus_muxer_select="ogg_muxer"
 psp_muxer_select="mov_muxer"
+rtc_muxer_deps_any="openssl"
```

Comment:
Derek Buitenhuis:

Further, I think this is missing a bunch of dependencies like HTTP and AVC?